grid_feats/roi_heads.py

```diff
@@ -226,12 +270,19 @@ def _forward_box(self, features, proposals):
 
             return losses
         else:
-            pred_instances, _ = self.box_predictor.inference(predictions, proposals)
-            return pred_instances
+            pred_instances, r_indices = self.box_predictor.inference(
+                predictions, proposals
+            )
+            return pred_instances[0], r_indices[0]
 
     def get_conv5_features(self, features):
         assert len(self.in_features) == 1
 
         features = [features[f] for f in self.in_features]
         return features[0]
 
+    def get_roi_features(self, features, proposals):
+        features = [features[f] for f in self.in_features]
+        box_features = self.box_pooler(features, [x.proposal_boxes for x in proposals])
+        fc7, fc6  = self.box_head(box_features)
+        return box_features, fc7, fc6
```

Comment:
I am not sure if it is trivial for users or not, but if we are storing fc6 anyway, might be worth saving the fc7 weights at the same time so that it can be directly loaded in mmf.